server/src/main/java/io/seata/server/ParameterParser.java

```diff
@@ -0,0 +1,116 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.server;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+
+/**
+ * The type parameter parser
+ *
+ * @author xingfudeshi@gmail.com
+ * @date 2019/05/30
+ */
+public class ParameterParser {
+    private static final String PROGRAM_NAME = "sh seata-server.sh";
+    private static final int SERVER_DEFAULT_PORT = 8091;
+    private static final String SERVER_DEFAULT_BIND_IP = "0.0.0.0";
```

Comment:
Here I want to bind an external ip to 127 and 0.0.0.0 even if the user does not pass this parameter. If the user does not pass, the returned xid is 0.0.0.0:8091:xxxxx, and I want to see the ip address of the destination server.

Reply:
> Here I want to bind an external ip not 127 and 0.0.0.0 even if the user does not pass this parameter. If the user does not pass, the returned xid is 0.0.0.0:8091:xxxxx, and I want to see the ip address of the destination server.

Got it,will change it.

Reply:
> Here I want to bind an external ip not 127 and 0.0.0.0 even if the user does not pass this parameter. If the user does not pass, the returned xid is 0.0.0.0:8091:xxxxx, and I want to see the ip address of the destination server.

it's done.